lib/ansible/modules/packaging/language/r_pkg_cran.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2019, Daniel Chen <chendaniely@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
```

Comment:
Sanity tests are failing due to missing `__future__` import.
```suggestion

from __future__ import absolute_import, division, print_function
__metaclass__ = type
```